test/e2e/cronjob.go

```diff
@@ -319,6 +424,28 @@ func waitForNoJobs(c clientset.Interface, ns, jobName string, failIfNonEmpty boo
 	})
 }
 
+// Wait for a list of jobs to not exist by listing jobs explicitly.
+func waitForJobsNotExist(c clientset.Interface, ns string, jobs []*batchv1.Job) error {
```

Comment:
Suggestion: `waitForJobNotExist` we care about one, as stated before. That will also simplify the code, b/c you'll get single job by name.